package.json

```diff
@@ -83,6 +83,7 @@
     "d3-time-format": "^4.1.0",
     "date-fns": "^2.28.0",
     "history": "^4.9.0",
+    "immer": "^10.1.1",
```

Comment:
In the PR description, can you also explain a bit more on why we chose "immer" over other alternatives please?

That information will be valuable for anyone looking for origin of immer as dependency.

Reply:
Thanks for updating PR description.